packages/api-client/src/components/Server/ServerDropdown.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
-import { useWorkspace } from '@/store'
+import { useLayout } from '@/hooks/useLayout'
+import { useWorkspace } from '@/store/store'
```

Comment:
```suggestion
import { useLayout } from '@/hooks'
import { useWorkspace } from '@/store'
```

or should we keep import more specific in genearl?

Reply:
ok reading the rest of the code i guess we might!

Reply:
Yes we need to start importing from each file and not the index files. Just do it as you go. It will help Hans prepare for module: nodenext